### ISSUE TITLE
MultipleMapping Class error 

### ISSUE THREAD

**Issue by DavidGorlin** [eyurtsev/FlowCytometryTools#45]
Hello,

Using FlowCytometryTools, we have had a number of issues running this code. We tend to get the error, "collections' has no attribute 'MutableMapping'". 

When calling FCMeasurment, we get that the MutableMapping method in Measurment class is the culprit. 

"MeasurementCollection(collections.MutableMapping, BaseObject
collections' has no attribute 'MutableMapping'"

In some environments, I have circumvented this by importing abc directly from collections in the begining of the script, but I am not sure why that is working, and doesn't always suffice.  Have you experienced this issue in the past? It looks like this class has been updated before. 

For background information, we are trying to install this on differnt machines. Some are mac M1 chip, some are Intel, some are microsoft, some are linux ubuntu. It doesn't seam to cooperate with all OS. We also have trouple installing it in a jupyter notebook. 

We were wondering if the code needs to be updated with new version of python..

**Comment by eyurtsev** [comment]
Yeah it was written for older python versions and needs to be upgraded
